### DISCUSSION TITLE
Unusually high system resources usage

### DISCUSSION THREAD

**Discussion by biennvops** [MRColorR/money4band#106] [Q&A]
<img width="1168" alt="image" src="https://github.com/user-attachments/assets/f2fc08b6-c93a-4476-a8dd-8eaf8cda1e61">
Is it normal or not?

**Comment by MRColorR** [comment]
The only strange behavior is with the Gradient app. It should be similar to Grass in terms of base image requirements. @Masterofnoothing , could you please take a look at it?

**Reply by Masterofnoothing** [reply]
Alright

**Reply by Masterofnoothing** [reply]
Gradient naturally has high system usage

**Reply by MRColorR** [reply]
In that case, I propose moving the Gradient app to a smaller Money4Band resource group; perhaps we could start by moving it to large group instead of huge one.

**Reply by MRColorR** [reply]
gradient has been moved to big group , but img is unstable atm i will disable it until the image is stable again . other apps in m4b works as intended
